### ISSUE TITLE
Parse error if array in a query has a space after the open-bracket

### ISSUE THREAD

**Issue by kliuless** [rmosolgo/graphql-ruby#68]
If I execute this query:

```
query test1 {
  testMe(array: [ "one"])
}
#                ^--- this space causes error
```

I get this error:
```
GraphQL::ParseError (Extra input after last repetition at line 1 char 1.)
```

If I remove the space after the open-bracket, the query succeeds.

**Comment by rmosolgo** [comment]
Thanks for reporting this! I think it's fixed by 655ae6e, could you give it a try?

**Comment by kliuless** [comment]
Thanks, confirmed fixed in f9bcce0.
